cpp/src/slice2py/PythonUtil.h

```diff
@@ -77,6 +81,27 @@ namespace Slice::Python
         std::string code;
     };
 
+    struct CompilationResult
+    {
+        /// The status of the compilation.
+        int status = EXIT_SUCCESS;
```

Comment:
(minor) In C++, we actually prefer `int status{EXIT_STATUS};`

But that's not the style we currently use in this file.